paddle/fluid/pybind/pybind.cc

```diff
@@ -775,7 +777,12 @@ All parameter, weight, gradient are variables in Paddle.
           })
       .def("set_int", [](ir::Pass &self, const std::string &name,
                          int val) { self.Set<const int>(name, new int(val)); })
-      .def("type", &ir::Pass::Type);
+      .def("type", &ir::Pass::Type)
+      .def("apply", [](ir::Pass &self, std::shared_ptr<ir::Graph> graph) {
+        std::unique_ptr<ir::Graph> origin_graph(graph.get());
```

Comment:
here `graph` should release because Apply can delete the graph.
you can reset `graph` after apply.

Reply:
fixed.